app/controllers/collections_controller.rb

```diff
@@ -43,16 +41,25 @@ def update
 
     @form = collection_form(collection)
     if @form.validate(collection_params) && @form.save
-      # TODO: https://github.com/sul-dlss/happy-heron/issues/92
-      # DepositCollectionJob.perform_later(@collection) if deposit_button_pushed?
-      redirect_to dashboard_path
+      collection.update!
```

Comment:
Can you say more about what this line is doing?

Reply:
It's a state transition... maybe `update` is a bad name for an event.

Reply:
Oh! Yes, would you mind renaming?

Reply:
renamed to `update_metadata`

Reply:
Thank you, sir.